packages/telestion-client-common/src/components/pages/dashboard-page/dashboard-page.tsx

```diff
@@ -6,6 +6,45 @@ import { Dashboard } from './dashboard/dashboard';
 import { NoDashboardsMessage } from './no-dashboards-message';
 import { useCurrentDashboards } from '../../../hooks';
 
+/**
+ * A Telestion Client page that renders a dashboard page.
+ * It displays the currently active dashboard of an authenticated user.
+ *
+ * It can be changed with the {@link DashboardPicker} component
+ * usually used in the application header.
+ * To register new dashboard for the user or change existing,
+ * please use the {@link useUserConfig} hook.
+ *
+ * It only renders if someone is authenticated.
+ * Otherwise, it redirects to the login page.
+ *
+ * **Attention**
+ *
+ * If you create your own dashboard page, please pass through
+ * the routing information so the pages component read it
+ * and render the page the right way:
+ * `MyDashboardPage.routing = TCDashboardPage.routing;`
```

Comment:
```suggestion
 * If you create your own dashboard page, please pass
 * the routing information so that the pages component read it
 * and render the page the right way:
 * `MyDashboardPage.routing = TCDashboardPage.routing;`
```